larray/core/session.py

```diff
@@ -1061,40 +1061,84 @@ def summary(self, template=None):
 
         Parameters
         ----------
-        template: str
+        template: dict {str: str}
             Template describing how items are summarized (see examples).
-            Available arguments are 'name', 'axes_names' and 'title'
+            Accepted keys are 'axis', 'group' and 'array'.
```

Comment:
wouldn't it be better to use directly the type as key instead of string, so that it is future proof (and users can add their own objects in the session and use summary)? eg:
print(s.summary({LArray: "{name} -> {axes_names}"}))